core/store/src/lib.rs

```diff
@@ -539,12 +539,12 @@ impl fmt::Debug for StoreUpdate {
         writeln!(f, "Store Update {{")?;
         for op in self.transaction.ops.iter() {
             match op {
-                DBOp::Insert { col, key, .. } => writeln!(f, "  + {col} {}", to_base58(key))?,
-                DBOp::Set { col, key, .. } => writeln!(f, "  = {col} {}", to_base58(key))?,
+                DBOp::Insert { col, key, .. } => writeln!(f, "  + {col} {}", pretty::Bytes(key))?,
+                DBOp::Set { col, key, .. } => writeln!(f, "  = {col} {}", pretty::Bytes(key))?,
```

Comment:
I woulder if we can base formatting on `col` rather than guessing? I *think* posvyatokum  introduces some key-types infra in a sibling PR? 

Unrelated to the PR, but I've also wondered if each col should actually *know* the full type of data stored there? It has a whole bunch of good properties, but one bad: that means that the store needs to be aware of hashes, blocks, and whatever else types on the higher layers of application.

Reply:
> I woulder if we can base formatting on col rather than guessing?

There’s also a question of whether it’s really worth the additional code.  If we had that information easily available it would be great but for now I’d stick to something more crude and simple.

> I think posvyatokum introduces some key-types infra in a sibling PR?

Yes, but it’s not clear whether we’re going to end up with that particular piece of code.  Furthermore, that code only cared about columns which go into cold storage so its accuracy might be tricky (similarly to how DBCol::is_col wasn’t correct).

> Unrelated to the PR, but I've also wondered if each col should actually know the full type of data stored there? It has a whole bunch of good properties, but one bad: that means that the store needs to be aware of hashes, blocks, and whatever else types on the higher layers of application.

I think doing that on Store level would make sense.  Or have some kind of `Store<Scheme>` type with generic magic which would allow access to subset of columns defined in the scheme.  But that’s really for future work.

Reply:
>I think doing that on Store level would make sense. 

Thinking out loud more, i do find something like that quite appealing. Specifically, if the store crate knows full schma, and deals with `BlockV1/V2/V3` stuff, and things like chain convert from that raw schema to more abstract domain objects. But that's indeed flying-cars type of future :)